llarp/router/router.cpp

```diff
@@ -711,6 +696,24 @@ namespace llarp
 
     if (bootstrapRCList.empty() and not conf.bootstrap.seednode)
     {
+#ifdef BOOTSTRAP_FALLBACK
```

Comment:
I would rather keep the `ifdef` as small as possible to minimize code compilation differences, perhaps via this outside the function:
```C++
static constexpr std::optional<std::string_view> bootstrap_fallback_path{
#ifdef BOOTSTRAP_FALLBACK
    std::string_view{BOOTSTRAP_FALLBACK}
#else
    std::nullopt;
#endif
};
```
Then the code here can do (without an ifdef):
```C++
if (bootstrap_fallback_path)
{
  ...
}
```